packaging/helm-charts/helm3/Makefile

```diff
@@ -25,6 +26,8 @@ helm_template:
 	# Generating rendered template files to: $(CHART_RENDERED_TEMPLATES_TMP)
 	mkdir -p $(CHART_RENDERED_TEMPLATES_TMP)
 	$(HELM_CLI) template --namespace myproject --output-dir $(CHART_RENDERED_TEMPLATES_TMP) $(CHART_PATH)
+	mkdir -p $(CHART_PATH)/files/grafana-dashboards
+	cp $(DASHBOARD_PATH)/*.json $(CHART_PATH)/files/grafana-dashboards
```

Comment:
This is good, but I think it is a bit confusing to have it here:
* This is step that actually gets things out of the Helm Chart. The lines above this already expect the Helm Chart to be ready -> so you essentially modify the Helm Chart after it is already used
* Having it here means it would be a different way than the other parts doing the same 

I would do the following:
* Add a deletion first to make sure any removed dashbaords are removed here as well
* I would move these lines as `dashboard_install` into the `packaging/examples/Makefile`
* And into the main `Makefile`, I would
    * Add this as a new line:
      ```
      dashboard_install: packaging/examples
      ```
    * Add it to the `all: ...` line after the `crd_install` step
* Add a new line to [this](https://github.com/strimzi/strimzi-kafka-operator/blob/main/.azure/scripts/uncommitted-changes.sh#L22) file with the `make dashboard_install` command as well

Reply:
I've taken a stab at this, but I'm only like 40% sure I got it right...

Reply:
I think it seems to be exactly as I would have done it ... so it has to be right :-D